bibliopixel/animation/animation.py

```diff
@@ -71,6 +104,7 @@ def stamp():
     @contextlib.contextmanager
     def run_context(self):
         self.completed = False
+        self.complete_reason = COMPLETE_REASON.NONE
```

Comment:
True... that could work. Again though, I don't like changing things out of the scope of this change. That should be another PR